.gitignore

```diff
@@ -35,3 +35,7 @@ e2e/.dev/mpa
 .husky/post-commit
 .husky/pre-push
 .husky/post-merge
+
+# For bison generated files used by ShaderLab
+*.tab.c
+*.output
```

Comment:
Try add how to debug with bison in ReadMe.md

Reply:
done